bower.json

```diff
@@ -9,14 +9,14 @@
   "license": "BSD-3-Clause",
   "repository": {
     "type": "git",
-    "url": "git://github.com/paf31/purescript-with-index.git"
+    "url": "https://github.com/paf31/purescript-with-index.git"
```

Comment:
The repository URL was updated to match the one in the [registry](https://github.com/purescript/registry/blob/master/bower-packages.json#L1407) so that a new version of the package can be published.